test/with_dummyserver/test_https.py

```diff
@@ -858,6 +858,26 @@ def test_warning_for_certs_without_a_san(self, no_san_server):
                 assert r.status == 200
                 assert warn.called
 
+    def test_common_name_without_san_with_different_common_name(
+        self, no_san_server_with_different_commmon_name
+    ):
+        ctx = util.ssl_.create_urllib3_context()
+        try:
```

Comment:
Since this is 1.26.x I think we're always checking commonName?

Reply:
Yes, You are right. Fixed!